src/metabase/core.clj

```diff
@@ -31,7 +34,16 @@
              [params :refer [wrap-params]]
              [session :refer [wrap-session]]]
             [toucan.db :as db])
-  (:import org.eclipse.jetty.server.Server))
+  (:import java.util.Locale
+           org.eclipse.jetty.server.Server))
+
+;; HACK: replace this with setting event listener
+(defn sync-locale []
+  (let [new-locale     (or (not-empty (setting/get :site-locale)) "en")
```

Comment:
I think since you set a default, if there's no value it will already be "en"

Reply:
Hmm you're right. For some reason I was getting null at some point. Maybe I accidentally set the value to null.

Reply:
Fixed